spec/content_tagger/adding_taxon_to_external_content_spec.rb

```diff
@@ -0,0 +1,46 @@
+feature "Adding a taxon to external content", new: true, collections: true, content_tagger: true do
+  include ContentTaggerHelpers
+  include PublisherHelpers
+
+  let(:taxon_title) { "Tagging external content taxon " + SecureRandom.uuid }
```

Comment:
Just an idea but we could have some global helpers which are like `unique_title` and `unique_slug` which suffix the uuid on the end so these can be a little more succinct as the implementation is very common. I imagine if you agree it'd be a separate PR job though. What do you think?

Reply:
Yeah I can see the value in this but I'm a fan of how explicit it is currently, could discuss later and raise it as a separate PR after if we think it's going to be valuable